front/app/components/LandingPages/citizen/TwoColumnLayout.ts

```diff
@@ -17,12 +17,29 @@ export const Container = styled.div`
   `}
 `;
 
-export const HeaderImage = styled(Image)`
-  height: ${homepageBannerLayoutHeights.two_column_layout.desktop}px;
+export const HeaderImageWrapper = styled.div`
+  width: 50%;
```

Comment:
Should we use the Box component here instead of a div? :slightly_smiling_face:  I can't quite remember what we agreed on here in the FE chapter :thinking: